bascule/basculehttp/enforcer.go

```diff
@@ -6,8 +6,28 @@ import (
 	"github.com/Comcast/comcast-bascule/bascule"
 )
 
+type NotFoundBehavior int
+
+// Behavior on not found
+const (
+	Forbid NotFoundBehavior = iota
+	Allow
+)
+
+func (n NotFoundBehavior) String() string {
```

Comment:
These strings are kinda long.  It's typical to use the name of the constant in code, e.g. "Forbid".

The golang way this is usually done is with the `stringer` generator.  See https://godoc.org/golang.org/x/tools/cmd/stringer.

The basic steps to use `stringer` are:

1. Put a go:generate comment in an obvious place.  Usually, in the same file as the type.  This is usually of the form `//go:generator stringer -type=NotFoundBehavior`
2. Make sure you have `stringer` installed in your workspace with `go get golang.org/x/tools/cmd/stringer`
3. Run `go generate` at the command line.

Steps 2 and 3 only need to be done the first time you create a type and any time you add values to the type.